### DISCUSSION TITLE
Is there a shortcut key for the global launcher?

### DISCUSSION THREAD

**Discussion by liby** [community/community#143914] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

In the [Gif demonstration](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot), I noticed this feature similar to a global launcher (very much like Raycast). Is it just the focusing effect of the introduction video?

<img width="819" alt="image" src="https://github.com/user-attachments/assets/47d677af-7c69-4d39-ab75-e0f0f9881f45">

![Source Gif](https://github.com/microsoft/vscode-copilot-release/blob/main/images/readme-gif.gif?raw=true)

**Comment by swimmingbrain** [comment]
Hey liby! In Windows 11, you can launch Microsoft Copilot using the keyboard shortcut Windows + C. This opens Copilot as a sidebar on the right edge of your screen, providing quick access to its features.

You can also use tools like PowerToys to remap an existing key to launch Copilot. For example, you can assign the 'Apps/Menu' key to trigger the Windows + C shortcut, effectively creating a single-key launcher for Copilot.

Check this link for some more details: https://allthings.how/how-to-assign-a-microsoft-copilot-key-on-any-keyboard-in-windows-11/

Hope this helps! 🧠

**Reply by liby** [reply]
I searched for Microsoft Copilot, and it looks different from what I saw.
![image](https://github.com/user-attachments/assets/885e9767-4a9b-4c81-b2e7-c7e11b8c13b0)

**Reply by swimmingbrain** [reply]
Oh, I see what you were asking now. Yes, that is simply a video effect used from them to zoom in and highlight that "chat" bar.

**Comment by amirinsight** [comment, marked as answer]
That is certainly just a video effect.
